File: server/router/admin/gost_client_logger.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/gost_client_logger"
	"server/global"
	"server/router/middleware"
)

func InitGostClientLogger(group *gin.RouterGroup) {
	g := group.Group("gost/client/logger", middleware.Auth(global.Jwt))
	g.POST("page", gost_client_logger.Page)
}

File: web/src/api/admin/gost_client_logger.js
import request from "../../request/index.js";

const baseUrl = '/v1/admin/gost/client/logger'

export const apiAdminGostClientLoggerPage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}
